komodo-shell/src/main/java/org/komodo/shell/commands/SetGlobalPropertyCommand.java

```diff
@@ -67,14 +69,25 @@ protected CommandResult doExecute() {
             String propValueArg = requiredArgument(1, I18n.bind(ShellI18n.invalidArgMsgPropertyValue ) );
 
             // validate global property name and value
-            final String errorMsg = getWorkspaceStatus().validateGlobalPropertyValue( propNameArg, propValueArg );
+            WorkspaceStatus wsStatus = getWorkspaceStatus();
+            String errorMsg = null;
+            if(wsStatus.isGlobalProperty(propNameArg)) {
+                errorMsg = getWorkspaceStatus().validateGlobalPropertyValue( propNameArg, propValueArg );
+            } else if(wsStatus.isProvidedGlobalProperty(propNameArg)) {
+                errorMsg = wsStatus.validateProvidedGlobalPropertyValue(propNameArg, propValueArg);
+            }
```

Comment:
Should we set an `errorMsg` when `propNameArg` is neither a global or a provided property? It seems there might be some issues later on in in lines 88-89 maybe. Or do you want a new provided global property to be created?